src/main/java/com/puppetlabs/jenkins/plugins/puppetgatling/gatling/PuppetGatlingBuildAction.java

```diff
@@ -67,7 +67,15 @@ public ReportRenderer getReport(String simulationName) {
 	}
 
 	public String getReportURL(String simulationName) {
-		return new StringBuilder().append(URL_NAME).append("/report/").append(simulationName).toString();
+		return new StringBuilder().append("job/").
```

Comment:
Yeah, I would have preferred to do it that way, but I couldn't.  The problem is that the link this was returning before was a relative URL, so the browser just adds it to whatever the current URL is.  That was broken because we have two different pages that can end up calling this, and those two pages are at different depths in the URL hierarchy.  So there is no single relative path that will work from both of those pages.

So, instead, I thought it was better to make the URL absolute.  So this PR changes the Jelly files to make them insert the 'root_url' at the beginning of the link, and then we know that this method should always return a path relative to the root Jenkins URL, and it will work from anywhere.